vulnerabilities/tests/test_mozilla.py

```diff
@@ -0,0 +1,97 @@
+import os
+import shutil
+import tempfile
+import zipfile
+from unittest.mock import patch
+
+from django.test import TestCase
+
+from vulnerabilities import models
+from vulnerabilities.import_runner import ImportRunner
+from vulnerabilities.importers.npm import categorize_versions
+
+BASE_DIR = os.path.dirname(os.path.abspath(__file__))
+TEST_DATA = os.path.join(BASE_DIR, "test_data/")
+
+
+@patch("vulnerabilities.importers.MozillaDataSource._update_from_remote")
+class MozillaImportTest(TestCase):
+
+    tempdir = None
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.tempdir = tempfile.mkdtemp()
+        zip_path = os.path.join(TEST_DATA, "mozilla.zip")
```

Comment:
Please avoid zip fixtures unless really needed, e.g. extremely rarely if ever

Reply:
This was inspired by npm and rust test cases. Removing it now.

Reply:
If we have zip in npm and rust we need to replace them too then... Do you mind to create a ticket for this>

Reply:
Created https://github.com/nexB/vulnerablecode/issues/442

Reply:
@Hritik14 npm and rust are for git ... yours are just plain text, so please do not zip here.